tools/build.gradle.kts

```diff
@@ -24,10 +27,30 @@ plugins {
   `cel-conventions`
 }
 
+apply<ProtobufPlugin>()
+
 dependencies {
   api(project(":cel-core"))
 
   testImplementation(platform(libs.junit.bom))
   testImplementation(libs.bundles.junit.testing)
   testRuntimeOnly(libs.junit.jupiter.engine)
 }
+
+// *.proto files taken from https://github.com/googleapis/googleapis/ repo, available as a git
+// submodule
```

Comment:
not sure i understand what the comment is trying to tell me, what proto files are we referring to here?

Reply:
copy-paste-mess